docs/permissions/writing-a-policy--new.md

```diff
@@ -0,0 +1,190 @@
+---
+id: writing-a-policy--new
+title: Writing a permission policy
+description: How to write your own permission policy as a Backstage integrator
+---
+
+In the [previous section](./getting-started--new.md), how to setup and enable the permissions system to prepare for writing a policy.
+
+In this section, we will learn how to write our own permission policy. In general terms, a policy is simply an async function that receives a request to authorize a specific action for a user and (optional) resource, and returns a decision on whether to authorize that permission. Integrators can implement their own policies from scratch, or adopt reusable policies written by others.
+
+A policy could look like this:
+
+```typescript title="packages/backend/src/plugins/permissions/policy.ts"
```

Comment:
I think I am going to change this to `packages/backend/src/plugins/permission.ts` that way its in-line with the rest of the documentation flow.